payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt

```diff
@@ -468,6 +472,29 @@ class LpmRepository constructor(
             requirement = AuBecsDebitRequirement,
             formSpec = LayoutSpec(sharedDataSpec.fields)
         )
+        PaymentMethod.Type.BacsDebit.code -> {
+            val localFields = listOf(
```

Comment:
Does this look right when the merchant configures billing address collection?

Reply:
Looks like no. If a merchant requests contact information (email/phone), it'll show up again at the end of the form.